src/Umbraco.Web.UI.Client/src/views/common/dialogs/iconpicker.html

```diff
@@ -1,63 +1,84 @@
-        		<option value="color-blue">Blue</option>	
-        		<option value="color-red">Red</option>
-        	</select>
-<!--
-        	<select class="input-block-level">
-        		<option value="">Black</option>
-        		<option value="color-green">Green</option>	
-        		<option value="color-blue">Turquoise</option>	
-        		<option value="color-purple">Purple</option>	
-        		<option value="color-red">Red</option>	
-        		<option value="color-orange">Orange</option>	
-        		<option value="color-sky-blue">Sky blue</option>	
-        		<option value="color-yellow">Yellow</option>		
-        	</select>-->
+            <select ng-model="color" class="input-block-level">
+                <option value="color-black">Black</option>
+                <option value="color-blue-grey">Blue Grey</option>
+                <option value="color-grey">Grey</option>
+                <option value="color-brown">Brown</option>
+                <option value="color-blue">Blue</option>
+                <option value="color-light-blue">Light Blue</option>
+                <option value="color-cyan">Cyan</option>
+                <option value="color-green">Green</option>
+                <option value="color-light-green">Light Green</option>
+                <option value="color-lime">Lime</option>
+                <!--<option value="color-yellow">Yellow</option>-->
+                <option value="color-amber">Amber</option>
+                <option value="color-orange">Orange</option>
+                <option value="color-deep-orange">Deep Orange</option>
+                <option value="color-red">Red</option>
+                <option value="color-pink">Pink</option>
+                <option value="color-purple">Purple</option>
+                <option value="color-deep-purple">Deep Purple</option>
+                <option value="color-indigo">Indigo</option>
+            </select>
+            <!--<select ng-model="color" class="input-block-level">
+                <option value="">Black</option>
+                <option value="color-green">Green</option>
+                <option value="color-yellow">Yellow</option>
+                <option value="color-orange">Orange</option>
+                <option value="color-blue">Blue</option>
+                <option value="color-red">Red</option>
+            </select>-->
+
+            <!--<select class="input-block-level">
+                <option value="">Black</option>
+                <option value="color-green">Green</option>
+                <option value="color-blue">Turquoise</option>
+                <option value="color-purple">Purple</option>
+                <option value="color-red">Red</option>
+                <option value="color-orange">Orange</option>
+                <option value="color-sky-blue">Sky blue</option>
+                <option value="color-yellow">Yellow</option>
+            </select>-->
 
         </div>
 
         <div class="umb-loader" ng-hide="icons"></div>
```

Comment:
Oh, I think we can change this line to this to prevent horizontal scroll:

```
<div class="umb-loader-wrapper" ng-hide="icons">
     <div class="umb-loader"></div>
</div>
```